src/app/component/auth/auth.module.ts

```diff
@@ -1,7 +1,7 @@
 import { NgModule } from '@angular/core';
 import { NewSignUpComponent } from './new-sign-up/new-sign-up.component';
-import {SignInNewComponent} from './sign-in-new/sign-in-new.component';
-import {CoreModule} from '../core/core.module';
+import { SignInNewComponent } from './sign-in-new/sign-in-new.component';
```

Comment:
Minor: Please don't  make any changes to files that aren't related to your direct ticket